redis/commands/core.py

```diff
@@ -3255,6 +3256,39 @@ def bzpopmin(self, keys, timeout=0):
         keys.append(timeout)
         return self.execute_command("BZPOPMIN", *keys)
 
+    def bzmpop(
+        self,
+        timeout: float,
+        num_keys: int,
```

Comment:
num_keys should probably be numkeys - given the [help text](https://redis.io/commands/bzmpop)